FetchRequests/Sources/CWFetchableObject.swift

```diff
@@ -52,31 +33,16 @@ public protocol CWFetchableObjectProtocol: class, CWIdentifiable, CWRawDataRepre
     /// - returns: The entityID for a fetchable object
     static func entityID(from data: RawData) -> ID?
 
-    static var idKeyPath: KeyPath<KeyPathBase, ID> { get }
-    static var dataKeyPath: KeyPath<KeyPathBase, RawData> { get }
-    static var deletedKeyPath: KeyPath<KeyPathBase, Bool> { get }
+    /// Listen for changes to the underlying data of `self`
+    func observeDataChanges(_ handler: @escaping (Self) -> Void) -> CWInvalidatableToken
 
-    var observingUpdates: Bool { get set }
+    /// Listen for changes to whether `self` is deleted
+    func observeIsDeletedChanges(_ handler: @escaping (Self) -> Void) -> CWInvalidatableToken
 
-    static func rawDataIsIdentical(lhs: RawData, rhs: RawData) -> Bool
+    /// Enforce listening for changes to the underlying data of `self`
+    func listenForUpdates()
 }
 
 extension CWFetchableObjectProtocol {
-    static func rawDataIsIdentical(lhs: RawData?, rhs: RawData?) -> Bool {
-        if lhs == nil, rhs == nil {
-            return true
-        } else if let lhs = lhs, let rhs = rhs {
-            return rawDataIsIdentical(lhs: lhs, rhs: rhs)
-        } else {
-            return false
-        }
-    }
-}
-
-#if canUseRawData
-extension CWFetchableObjectProtocol where RawData: Equatable {
-    static func rawDataIsIdentical(lhs: RawData, rhs: RawData) -> Bool {
-        return lhs == rhs
-    }
+    public func listenForUpdates() {}
```

Comment:
Should we actually be providing a default implementation for this method? This seems like a method developers should need to be forced to implement

Reply:
I was guessing that a lot of frameworks would automatically just do this by default?
Like if I had built CWObject initially I wouldn't have required opting into live updates.

Reply:
Sounds good to me